secboot/secboot_hooks.go

```diff
@@ -98,19 +97,14 @@ func SealKeysWithFDESetupHook(runHook fde.RunSetupHookFunc, keys []SealKeyReques
 		if primaryKey == nil {
 			primaryKey = primaryKeyOut
 		}
-		if err := skr.BootstrappedContainer.AddKey(skr.SlotName, unlockKey); err != nil {
+		tokenWriter, err := skr.BootstrappedContainer.AddKeyAndGetTokenWriter(skr.SlotName, unlockKey)
```

Comment:
Oops, I forgot to stilll  use key file if KeyFile was not empty here. I will push an update soon.